app/src/main/java/com/vero/tasky/agenda/presentation/components/TimePickerLabelComponent.kt

```diff
@@ -0,0 +1,48 @@
+package com.vero.tasky.agenda.presentation.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowRight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.vero.tasky.R
+import com.vero.tasky.ui.theme.Inter400Size16
+
+@Composable
+fun TimePickerLabelComponent(
+    label: String,
+    value: String,
+    isEditable: Boolean,
+    onEditClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Box(modifier = modifier.width(120.dp).clickable { if (isEditable) onEditClick() }) {
```

Comment:
I think the clickable modifier has an enabled value to enable or disable the click lambda. This would be better here since with your version you still get a ripple effect, but nothing will happen if the user is not in edit mode which could be confusing